test cases/python/9 extmodule limited api/test_limited.py

```diff
@@ -0,0 +1,4 @@
+from limited import hello
+
+def test_hello():
+    assert hello() == "Hello, world"
```

Comment:
What's the point of the assert if it is not executed in the test? It is no executed because the test function returns `hello world` not `Hello, world` and the test would fail. If the point is to check that `from limited import hello` works, there is no need to define the `test_hello()` function. At that point, you can make the test definition in `meson.build` simply execute `python -m limited.hello`.

Reply:
So, I'm implementing what you suggested and in the process have learned the `python -m foo.bar` does not work for modules that are implemented as C extensions. This was news to me, so instead what I propose to do instead is to simply fix my original broken test such that `test_hello` is actually called, and that it actually checks for the right value. Does that sound ok?

Reply:
That would have been my suggestion anyway, yeah.